policy-design-lab/pdl-frontend
# Update ira page message modal and update EQIP & CSP page for new data

This PR moves the IRA message to the on-page setting and updates the EQIP and CSP pages.

### How to Test

1. Navigate to https://policydesignlab-dev.ncsa.illinois.edu/ira. You should not see the information modal appear when the page finishes loading. Instead, you should see a panel at the top of the page containing the first few sentences of the information modal.
![Screenshot 2024-08-13 at 3 03 57 PM](https://github.com/user-attachments/assets/80102d13-c932-484e-afe2-7904bd1797d7)

2. Click the 'LEARN MORE' button at the end of the paragraph. You should be able to read all the introduction messages in the modal.

3. Navigate to https://policydesignlab-dev.ncsa.illinois.edu/eqip and https://policydesignlab-dev.ncsa.illinois.edu/csp. You should see that all of the components have updated data.
